packages/l/libsigrokdecode/package.yml

```diff
@@ -1,18 +1,20 @@
 name       : libsigrokdecode
 version    : 0.5.3
-release    : 9
+release    : 10
 source     :
     - https://sigrok.org/download/source/libsigrokdecode/libsigrokdecode-0.5.3.tar.gz : c50814aa6743cd8c4e88c84a0cdd8889d883c3be122289be90c63d7d67883fc0
+homepage   : https://sigrok.org
 license    : GPL-3.0-or-later
 component  : programming.library
 summary    : Provides the basic API for sigrok protocol decoders
 description: |
     libsigrokdecode is a shared library written in C which provides the basic API for running sigrok protocol decoders. The protocol decoders themselves are written in Python.
 builddeps  :
     - pkgconfig(python3)
+    - pkgconfig(python3-embed)
```

Comment:
This is already provided as a part of the `python3` package.

Reply:
Thanks. Please take another look now.